src/toTransform.js

```diff
@@ -203,63 +209,42 @@
       return null;
     }
 
-    if (!position || position === 'auto') {
-      var result = {
-        deltaX: 0,
-        deltaY: 0,
-        transformOriginX: transformOrigin[0].value,
-        transformOriginY: transformOrigin[1].value,
-        offsetPosX: offsetLeft,
-        offsetPosY: offsetTop,
-        containerWidth: parentProperties.width,
-        containerHeight: parentProperties.height
-      };
-      if (!anchor || anchor === transformOrigin || anchor === 'auto') {
-        result['anchorX'] = transformOrigin[0].value;
-        result['anchorY'] = transformOrigin[1].value;
-        return result;
-      }
+    var anchorPosX;
+    var anchorPosY;
+    if (anchor === 'auto') {
+      anchorPosX = transformOrigin[0].value;
+      anchorPosY = transformOrigin[1].value;
+    } else {
+      anchorPosX = anchor[0].value;
+      anchorPosY = anchor[1].value;
       if (anchor[0].unit === '%') {
-        result['anchorX'] = (anchor[0].value * elementProperties.width) / 100;
-      } else {
-        result['anchorX'] = anchor[0].value;
+        anchorPosX = (anchorPosX * elementProperties.width) / 100;
       }
 
       if (anchor[1].unit === '%') {
-        result['anchorY'] = (anchor[1].value * elementProperties.height) / 100;
-      } else {
-        result['anchorY'] = anchor[1].value;
+        anchorPosY = (anchorPosY * elementProperties.height) / 100;
       }
-      return result;
-    }
-
-    var anchorPosX = anchor[0].value;
-    var anchorPosY = anchor[1].value;
-    var offsetPosX = position[0].value;
-    var offsetPosY = position[1].value;
-
-    if (anchor[0].unit === '%') {
-      anchorPosX = (anchorPosX * elementProperties.width) / 100;
     }
 
-    if (anchor[1].unit === '%') {
-      anchorPosY = (anchorPosY * elementProperties.height) / 100;
-    }
-
-    if (position[0].unit === '%') {
-      offsetPosX = (offsetPosX * parentProperties.width) / 100;
-    }
-
-    if (position[1].unit === '%') {
-      offsetPosY = (offsetPosY * parentProperties.height) / 100;
+    var offsetPosX;
+    var offsetPosY;
+    if (position === 'auto') {
+      offsetPosX = offsetLeft + anchorPosX;
```

Comment:
this seems to be a behavior change from before when a position of auto resulted in offsetPosX = offsetLeft and offsetPosY = offsetTop. Can you explain this change please?